starters/vue3-xstate-css/cypress.config.ts

```diff
@@ -3,10 +3,6 @@ import { defineConfig } from 'cypress';
 export default defineConfig({
   viewportHeight: 720,
   viewportWidth: 1280,
-  e2e: {
```

Comment:
Removed these as the testing focus was on component tests and not e2e. Didn't want to leave the setup for e2e when there are zero tests to run for it.